dcadec.pc.in

```diff
@@ -1,9 +1,10 @@
-prefix=%PREFIX%
-libdir=%LIBDIR%
-includedir=%INCLUDEDIR%
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
 
 Name: dcadec
 Description: a free DTS Coherent Acoustics decoder with support for HD extensions
-Version: %VERSION%
+Version: @version@
 Libs: -L${libdir} -ldcadec
-Cflags: -I${includedir}
+Cflags: -I${includedir}/dcadec
```

Comment:
This breaks compatibility.

1. Currently users can use `#include <libdcadec/dca_context.h>`. Now you are force them to use `#include <dca_context.h>`, when `${includedir}` is not already in the search path of the preprocessor.
2. Below you are using `pkginclude_HEADERS`. The headers will be installed to `dcadec/dca_context.h` rather than `libdcadec/` like it used to be.